watertap/data/techno_economic/tests/test_unit_parameter_files.py

```diff
@@ -52,6 +52,7 @@ def test_unit_parameter_files(tech):
                     "municipal_drinking",
                     "pump",
                     "storage_tank",
+                    "static_mixer"
```

Comment:
```suggestion
                    "static_mixer",
```
This isn't a syntax error as-it-is, but it would bite us eventually.